lib/rubocop/cop/minitest/refute_empty.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Check if your test uses `refute_empty` instead of `refute(actual.empty?)`.
+      #
+      # @example
+      #   # bad
+      #   assert(actual.empty?)
```

Comment:
I think this doc is not equal the behave you implemented 🤔 

In the code you are registering an offsense if using `refute(actual.empty?)` and here you are saying that the bad is `assert(actual.empty?)`. 

Also, it would be awesome if you can make the cop look for both: `refute(something.empty?)` and assert(!something.empty)`.

Reply:
@duduribeiro: My bad. I'll fix the documentation. Regarding adding `assert(!something.empty)`, I was planning to do it in another PR. Hope that is fine